superset/utils/decorators.py

```diff
@@ -69,3 +74,34 @@ def wrapped(*args: Any, **kwargs: Any) -> Any:
         return wrapped
 
     return decorate
+
+
+def on_security_exception(self, ex) -> Response:
+    return self.response(403, **{"message": utils.error_msg_from_exception(ex)})
+
+
+def check_permissions(
```

Comment:
The name of this decorator is too generic for a check that's only on dashboards.